docs/docs/view-components.md

```diff
@@ -120,14 +131,14 @@ It is up to the developer on how to handle this most efficiently, but here's an
 
       // Override `nitroId` setter to throw `this` into global map
       fun setNitroId(nitroId: Double) {
-        globalViewsMap.put(nitroId, WeakReference(this))
+        globalViewsMap[nitroId] = WeakReference(view)
```

Comment:
I mean, okay yea that also works I guess?

Reply:
Yes. it work. suggestion by Android Studio :V